### PR TITLE
lsn/ update doc

### PR DESCRIPTION
**Pull request by LeungChiNan**
## Motivation and Context
update diopi test doc

## Description
diopi readme
diopi_test readme

## Checklist
**Before PR**:

- [x] I have read and followed the workflow indicated in the [Contributors.md](https://github.com/DeepLink-org/DIOPI/blob/main/Contributors.md) to create this PR.
- [x] Pre-commit or linting tools indicated in [Contributors.md](https://github.com/DeepLink-org/DIOPI/blob/main/Contributors.md) are used to fix the potential lint issues.
- [x] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [x] New functionalities are covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [x] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [x] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

**Review by yangbofun — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -75,30 +75,39 @@ DIOPI主要包含以下几个组件：
...
     ```
-    python main.py --mode run_test
+    python main.py --mode gen_case  # 生成pytest测例
+    python main.py --mode run_test  # 执行测试
     ```
     如需指定模型：
     ```
-    python main.py --mode run_test --model_name xxx
+    python main.py --mode gen_case --model_name xxx
+    python main.py --mode run_test --file_or_dir /path/to/case
```
Comment (jingguo-st) [inline README.md:84]: file和dir分别给出一个例子，对参数说明更准确

File: README.md [hunk 2]
```diff
@@ -75,30 +75,39 @@ DIOPI主要包含以下几个组件：
 ## 校验算子
 1. 将数据拷贝到芯片机器上，执行以下命令验证算子：
```
Comment (jingguo-st) [inline README.md:76]: 说明data是哪里来的，在哪个平台上生成的，可以给出链接

Comment (jingguo-st) [inline README.md:76]: 说明data是哪里来的，在哪个平台上生成的，可以给出链接

File: README.md (hunk 1 above)
Comment (jingguo-st) [inline README.md:84]: --file_or_dir 改成--test_cases_path, 同步main.py内也修改一下

File: diopi_test/README.md [hunk 3]
```diff
@@ -330,6 +337,7 @@ DIOPI-TEST 设计了一套测例配置规则及相应的测试框架。以算子
 
 
 ```
-    python main.py --mode run_test --impl_folder /path/to/folder --fname cdist
+    python main.py --mode gen_case --impl_folder /path/to/folder --fname cdist
+    python main.py --mode run_test --path_or_dir /path/to/cdist/case
```
Comment (jingguo-st) [inline diopi_test/README.md:341]: 同步改一下

**Review by jingguo-st — APPROVED** [review]

**Pull request merged**